package.json

```diff
@@ -5,6 +5,9 @@
   "repository": "git@github.com:nikrolls/json-formatter.git",
   "author": "Nik Rolls <nik@rolls.cc>",
   "license": "MIT",
+  "scripts": {
+    "watch": "nodemon -i build -x \"gulp build:dist\""
```

Comment:
FYI, this is just a dev convenience to trigger a rebuild any time a file changes.  I have nodemon installed globally, so haven't added it as a devDependency here.